lib/hammer_cli/output/adapter/abstract.rb

```diff
@@ -55,13 +55,24 @@ def self.data_for_field(field, record)
         elsif record && record.kind_of?(Hash) && record.has_key?(path_key.to_s)
           record[path_key.to_s]
         else
-          return nil
+          # there is no way to recognize why nil is returned from this
+          # method. Raising signals that there is missing response from server,
+          # not just nil value is returned.
+          raise HammerCLI::MissingAPIError
         end
       end
     end
 
     def data_for_field(field, record)
-      Abstract.data_for_field(field, record)
+      begin
+        return Abstract.data_for_field(field, record)
+      rescue HammerCLI::MissingAPIError
+        if field.hide_missing? && (self.class != CSValues)
+          raise HammerCLI::MissingAPIError
```

Comment:
`rescue HammerCLI::MissingAPIError => e` and then `raise e`. Btw why is `#dafa_for_field` implemented differently from `.data_for_field` ? Shouldn't they behave the same?